sky_alert/endpoint.py

```diff
@@ -7,6 +14,73 @@
 def healthz():
     return 'OK'
 
-@app.get('/sunrise')
-def sunrise():
-    return '06:00'
\ No newline at end of file
+@app.get('/openweather_sun_data')
+def sun_data(lat: int = 0, lon: int = 0):
```

Comment:
Let's add the return type here in the method signature too 
```suggestion
def sun_data(lat: int = 0, lon: int = 0) -> SunData:
```